src/source.rs

```diff
@@ -18,7 +18,7 @@ use crate::ipmi::{Ipmi, SensorReading};
 /// SMART data does not include the temperature or if the reported temperature
 /// exceeds the bounds of a `u8`, then `Ok(None)` is returned.
 fn parse_smart_source<T: AsRef<Path>>(block_dev: T) -> Result<Option<u8>> {
-    let proc = Command::new("smartctl")
+    let mut proc = Command::new("smartctl")
         .arg("-j")
         .arg("-A")
```

Comment:
While we're here, curious why you might have landed on `-A` instead of the more targeted `-l scttempsts`? I was bench-marking these and found that -A was slower. Not significantly slower, just maybe 5-10%

Reply:
Good question. I did try `-l scttempsts`, but it seems to only return temperature data with my SATA3 drives. With my SAS3 and NVME drives, it returned no data. `-A` seems to work with all 3 types.

Reply:
Whoops, I only tried my SATA drives. Looking at the docs again, I missed an important note:

> scttemp, scttempsts, scttemphist - [ATA only]

Please ignore!